apps/ubuntu/resources.qrc

```diff
@@ -0,0 +1,43 @@
+<RCC>
```

Comment:
Ok, so mostly build related. I'll stick to normal files on Sailfish then.

I've also read about performance I think, but well, it isn't like the app or loading the QML is slow, so no need to update those for it.